### ISSUE TITLE
Check that buffer is correctly accessed in WHERE clause rule

### ISSUE THREAD

**Issue by Ton-Pe** [Riverside-Software/sonar-openedge#1126]
Rule:  BufferUsageInWhereClause 
Following find statement raises an issue: "WHERE clause does not have any reference to buffer ProdRes"

```
 FIND FIRST ProdOper OF ProdOrder WHERE
          CAN-FIND(FIRST ProdRes OF ProdOper WHERE  ProdRes.Qtyrep = YES).

```
Looks like that the rule does not recognize the OF statement that is used in the CAN-FIND part of the WHERE clause.

If I write the can-find without OF, issues is not raised
```
FIND FIRST ProdOper OF ProdOrder WHERE
          CAN-FIND(FIRST ProdRes WHERE ProdRes.PrOrderNum = ProdOper.PrOrderNum AND ProdRes.Qtyrep = YES).
```
